sections/07_conclusions.md

```diff
@@ -1,15 +1,60 @@
+disease?  Though the answer is highly dependent on the specific domain and
+problem being addressed, we conclude that deep learning has not *yet* realized
+its transformative potential or induced a strategic inflection point.  Despite
+its dominance over competing machine learning approaches in many of the areas
+reviewed here and quantitative improvements in predictive performance, deep
+learning has not yet qualitatively "solved" those problems that were previously
+"unsolved".
 
-Points to mention based on discussion thus far that may make the bar for
-conclusions:
+As an analogy, consider recent progress in conversational speech recognition.
+Since 2009 there have been drastic performance improvements, with error rates
+dropping from more than 20% to less than 6% [@tag:Speech_recognition] and
+finally approaching or exceeding human performance in the past year
+[@arxiv:1610.05256 @arxiv:1703.02136] `TODO: working on a second source for this
+error trajectory from a talk by Eric Horvitz`. The phenomenal  improvements on
+benchmark datasets are undeniable, but the successes of the early 2010s did not
+fundamentally transform the domain. `TODO: this is debatable, maybe need a
+different example or to clarify what is meant by "conversational" speech`
+Widespread adoption of these technologies will requires not only improvements
+over baseline methods but truly "solving" the problem, in this case exceeding
+human-level performance, as well as convincing users to embrace the technology
+[@tag:Speech_recognition].  We see parallels to the healthcare domain, where
+achieving the full potential of deep learning will require outstanding
+predictive performance as well as adoption by biologists and clinicians.
 
-* Limitations of data & workarounds (availability impacts on amount, etc)
-* Transferability of features
-* Strong enthusiasm for unsupervised approaches.
-* Right to an explanation (possibly, depends if raised in multiple areas)
+Some of the areas we have discussed are closer to surpassing this lofty bar than
+others, generally those that are more similar to the non-biomedical tasks that
+are now monopolized by deep learning.  In medical imaging, diabetic retinopathy
+[@doi:10.1001/jama.2016.17216], diabetic macular edema
+[@doi:10.1001/jama.2016.17216], and skin lesion [@doi:10.1038/nature21056]
+classifiers are highly accurate and comparable to dermatologist performance in
+the latter case. `TODO: more imaging examples or other examples that might be at
+or close to "transformative"?`  In other domains, perfect accuracy will not be
+required because deep learning will be used primarily to prioritize experiments.
+For example, in chemical screening for drug discovery, a deep learning system
+that successfully identifies dozens or hundreds of target-specific, active
+small molecules from a massive search space would have immense practical value
+even if its overall precision is modest. Conversely, the most challenging tasks
```

Comment:
Perhaps I mean to say that errors may be magnified in some cases if we rely on predictions for some secondary task.  For instance, suppose I rely on predictions of TF binding from a neural net to predict TF binding and then use those genome-wide predictions to model gene expression.  I'll need to be more accurate than if I were going to use those predictions to follow up on a few interesting or high-confidence binding sites.